packages/asc-ui/src/components/ImageCard/ImageCard.stories.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import styled from '@datapunt/asc-core'
+import { storiesOf } from '@storybook/react'
+import ImageCard from './ImageCard'
+import { ImageCardContent } from './ImageCardContent'
+import Heading from '../Heading'
+import Paragraph from '../Paragraph'
+import Row from '../Grid/Row'
+import Column from '../Grid/Column'
+import { breakpoint } from '../../utils'
+import { ImageCardWrapperStyle } from './ImageCardStyle'
+import TypographyStyle from '../Typography/TypographyStyle'
+
+const ImageCardConstainer = styled.section`
+  width: calc(100% + 24px);
+  margin-left: -12px;
+  margin-right: -12px;
+  display: flex;
+  flex-wrap: wrap;
+
+  @media screen and ${breakpoint('min-width', 'laptop')} {
+    width: calc(100% + 4px);
+  }
+`
+const ImageCardWrapperBig = styled.div`
+  flex-basis: 100%;
+  @media screen and ${breakpoint('min-width', 'tabletM')} {
+    flex-basis: ${100 - 100 / 3}%;
+  }
+`
+const ImageCardWrapperSmall = styled.div`
```

Comment:
these styles could be moved to separate files or, if that specific, maybe to atlas

Reply:
why? these are just used for the atlas implementation  story

Reply:
ok, so thet are to be implemented in atlas as well. nice!